server/controllers/refer/new/additionalInformationController.ts

```diff
@@ -60,12 +59,37 @@ export default class NewReferralsAdditionalInformationController {
         return res.redirect(authPaths.error({}))
       }
 
-      const formattedAdditionalInformation = req.body.additionalInformation?.trim()
+      const referralUpdate: ReferralUpdate = {
+        additionalInformation: null,
```

Comment:
This would (unintentionally) delete any previously entered additional information.